src/sw.test.js

```diff
@@ -58,24 +50,39 @@ describe('Service Worker', () => {
         expect(fetchSpy).toHaveBeenCalledWith(expectedUrl, { mode: 'no-cors' });
       });
     });
+    describe('image requested when webp supported', () => {
+      // The service worker doesn't do anything when webp is supported
+      it.each`
+        image
+        ${`${TEST_IMAGE_URL}/news/puppies.jpg.webp`}
+      `(`for $image is $expectedUrl`, async ({ image }) => {
+        ({ fetchEventHandler } = await import('./service-worker-test'));
+
+        const event = {
+          request: new Request(image, { headers: { accept: 'webp' } }),
+        };
+
+        event.respondWith = jest.fn();
+
+        await fetchEventHandler(event);
+
+        expect(event.respondWith).not.toHaveBeenCalled();
+      });
+    });
 
     describe('image is not requested', () => {
       it.each`
-        image                                                 | headers               | reason
-        ${`${TEST_IMAGE_URL}/sport/puppies.jpg`}              | ${{ accept: 'webp' }} | ${'image url must include news, ace/standard or ace/ws'}
-        ${`${TEST_IMAGE_URL}/ace/stndard/puppies.jpg`}        | ${{ accept: 'webp' }} | ${'image url must include news, ace/standard or ace/ws'}
-        ${`${TEST_IMAGE_URL}/ace/sw/puppies.jpg`}             | ${{ accept: 'webp' }} | ${'image url must include news, ace/standard or ace/ws'}
-        ${`${TEST_IMAGE_URL}/news/puppies.jpeg`}              | ${{ accept: 'webp' }} | ${'image extension must be jpg or png'}
-        ${`${TEST_IMAGE_URL}/news/amz/puppies.jpeg`}          | ${{ accept: 'webp' }} | ${'image url must not include amz'}
-        ${`${TEST_IMAGE_URL}/news/worldservice/puppies.jpeg`} | ${{ accept: 'webp' }} | ${'image url must not include worldservice'}
-        ${`${TEST_IMAGE_URL}/news/puppies.jpg`}               | ${{}}                 | ${`webp not supported in request headers`}
-        ${`${BASE_IMAGE_URL}/sport/puppies.jpg`}              | ${{ accept: 'webp' }} | ${'image url must include news, ace/standard or ace/ws'}
-        ${`${BASE_IMAGE_URL}/ace/stndard/puppies.jpg`}        | ${{ accept: 'webp' }} | ${'image url must include news, ace/standard or ace/ws'}
-        ${`${BASE_IMAGE_URL}/ace/sw/puppies.jpg`}             | ${{ accept: 'webp' }} | ${'image url must include news, ace/standard or ace/ws'}
-        ${`${BASE_IMAGE_URL}/news/puppies.jpeg`}              | ${{ accept: 'webp' }} | ${'image extension must be jpg or png'}
-        ${`${BASE_IMAGE_URL}/news/amz/puppies.jpeg`}          | ${{ accept: 'webp' }} | ${'image url must not include amz'}
-        ${`${BASE_IMAGE_URL}/news/worldservice/puppies.jpeg`} | ${{ accept: 'webp' }} | ${'image url must not include worldservice'}
-        ${`${BASE_IMAGE_URL}/news/puppies.jpg`}               | ${{}}                 | ${`webp not supported in request headers`}
+        image                                                      | headers               | reason
```

Comment:
I think we should remove the `headers` column from the table now - given that webp is the default, then we can probably hardcode it below. This is to try and get coverage for all other scenarios (and some of them may even be invalid now)